client/src/pages/Login/useStyles.ts

```diff
@@ -6,16 +6,29 @@ const useStyles = makeStyles(() => ({
     '& .MuiInput-underline:before': {
       borderBottom: '1.2px solid rgba(0, 0, 0, 0.2)',
     },
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#FAFAFA',
+  },
+  container: {
+    height: '100%',
+    background: '#fff',
+    borderRadius: 10,
+    boxShadow: '1px -1px 23px 2px rgba(218,225,240,0.75)',
+    webkitboxShadow: '1px -1px 23px 2px rgba(218,225,240,0.75)',
+    mozboxShadow: '1px -1px 23px 2px rgba(218,225,240,0.75)',
   },
   authWrapper: {
-    minHeight: '100vh',
     paddingTop: 23,
+    display: 'flex',
   },
   welcome: {
-    fontSize: 26,
+    fontSize: 10,
```

Comment:
Alternatively, we can assign a font size to different variants within MUI's theme and use those variants throughout the app to ensure a consistent font-size